plugins/npm/nexus-npm-repository-plugin/src/main/java/com/bolyuba/nexus/plugin/npm/service/internal/orient/OrientMetadataStoreLifecycle.java

```diff
@@ -53,7 +53,14 @@ public void on(final NexusInitializedEvent e) throws Exception {
 
   @Subscribe
   public void on(final NexusStoppedEvent e) throws Exception {
-    orientMetadataStore.stop();
-    Orient.instance().shutdown();
+    log.debug("Asking npm metadata store to stop...");
+    try{
+      orientMetadataStore.stop();
+    } catch( Exception ex ){
+      log.warn("Problem stopping the npm metadata store", ex);
+    } finally {
+       log.debug("Asking Orient to shutdown...");
```

Comment:
`Orient.instance().shutdown()` also contains logging calls at debug, so I don't think you need to worry about that.
(that said, not sure this log message adds much value given the existing logging in `Orient.instance().shutdown()`)